qml/EntityList.qml

```diff
@@ -278,8 +275,7 @@ Rectangle {
                                                         anchors.fill: parent
                                                         onDoubleClicked: {
                                                             controller.participant_click(id)
-                                                            updateLastEntityClicked(participantIdx, endpointIdx, locatorIdx)
-                                                            lastClickedPhysical()
+                                                            lastClickedDDSEntity(participantIdx, endpointIdx, locatorIdx, name, kind)
                                                         }
                                                     }
                                                 }
```

Comment:
_This same file in line 296:297_
Use Endpoint and Locator instead of User and Process